ide/utils/sdk/sdk_scripts.py

```diff
@@ -298,9 +298,9 @@ def build(ctx):
 
     ctx.set_group('bundle')
     ctx.pbl_bundle(binaries=binaries, js=ctx.path.ant_glob(['src/pkjs/**/*.js', 'src/pkjs/**/*.json']), js_entry_file='src/pkjs/{{pkjs_entry}}')
-"""
+""".replace('{{pkjs_entry}}', project.pkjs_entry_point)
```

Comment:
You mean, something like... 

```
return wscript.replace('{{jshint}}', 'True' if jshint and not for_export else 'False').replace('{{pkjs_entry}}', project.pkjs_entry_point or '')
```
?
It was chained before, but the problem was that `project.pkjs_entry_point` can be `None`.